styleguide/pane.scss

```diff
@@ -294,3 +294,61 @@ $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
   color: $green;
   margin: 4px 0; // vertically center with the icon (24px - 16px)
 }
+
+/* filtered item search pane (add component, possibly new page in the future) */
+.kiln-toolbar-pane .filtered-input {
+  @include input();
+
+  margin: 0;
+  position: relative;
+  width: 100%;
+}
+
+.kiln-toolbar-pane .filtered-items {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  width: 100%;
+}
+
+.kiln-toolbar-pane .filtered-item {
+  @include primary-text();
+
+  cursor: pointer;
+  line-height: 16px;
+  list-style: none;
+  margin: 0;
+  padding: 15px 0;
+  width: 100%;
+}
+
+.kiln-toolbar-pane .filtered-item.filtered {
+  display: none;
+}
+
+.kiln-toolbar-pane .filtered-item:not(.filtered) {
+  border-bottom: 1px solid $grey;
+  transition: 200ms border-bottom-color ease-out;
+}
+
+.kiln-toolbar-pane .filtered-item:focus {
+  border-bottom: 1px solid $blue;
+  outline: none;
+  transition: 200ms border-bottom-color ease-out;
+}
+
+.kiln-toolbar-pane .filtered-item.active {
+  border-bottom: 2px solid $blue;
+}
+
+@keyframes shake {
+  0%, 100% {transform: translateX(0);}
+  20%, 60% {transform: translateX(-5px);}
+  40%, 80% {transform: translateX(5px);}
+}
+
+.kiln-shake {
+  animation-name: shake;
+  animation-duration: 300ms;
+  animation-fill-mode: both;
+}
```

Comment:
This little animation really makes the tool feel alive (if if that illusion is only for a second.) Would be great to think about how else to continue high-touch details.